frontend/package.json

```diff
@@ -1,60 +1,59 @@
 {
-  "name": "dashboard",
-  "version": "0.0.1",
+  "name": "sa-dashboard",
+  "version": "2.0.0",
   "private": true,
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "5.0.0",
+    "sass": "^1.49.7",
+    "web-vitals": "^2.1.4"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "husky": "^7.0.4",
+    "lint-staged": "^12.3.3",
+    "prettier": "^2.5.1",
+    "pretty-quick": "^3.1.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
+      "prettier --write"
+    ]
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "lint": "eslint --ignore-path ../.gitignore --ext .js,.jsx,.ts,.tsx ."
-  },
-  "dependencies": {
-    "@mapbox/mapbox-sdk": "^0.12.1",
-    "@popperjs/core": "^2.9.2",
-    "@turf/turf": "^6.3.0",
-    "downshift": "^6.1.2",
-    "lru-cache": "^6.0.0",
-    "mapbox-gl": "^2.7.0",
-    "mathjax-react": "^1.0.6",
-    "popper-max-size-modifier": "^0.2.0",
-    "prop-types": "^15.7.2",
-    "rc-slider": "^9.7.2",
-    "rc-table": "^7.15.1",
-    "react": "^16.9.0",
-    "react-collapsible": "^2.8.3",
-    "react-dom": "^16.9.0",
-    "react-popper": "^2.2.5",
-    "react-redux": "^7.2.3",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
-    "react-tooltip": "^4.2.17",
-    "recharts": "^2.0.9",
-    "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0",
-    "topojson-client": "^3.1.0",
-    "topojson-server": "^3.0.1",
-    "topojson-simplify": "^3.0.3",
-    "use-debounce": "^6.0.1"
+    "lint": "eslint --ignore-path ../.gitignore --ext .js,.jsx,.ts,.tsx .",
+    "lint-fix": "eslint --ignore-path ../.gitignore --ext .js,.jsx,.ts,.tsx . --fix",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "prettier": "pretty-quick --staged",
+    "prepare": "cd .. && husky install frontend/.husky"
   },
   "eslintConfig": {
     "extends": [
-      "react-app"
+      "react-app",
+      "react-app/jest"
     ]
   },
   "browserslist": {
     "production": [
       ">0.2%",
       "not dead",
-      "not op_mini all",
-      "not chrome < 51",
-      "not safari < 10"
+      "not op_mini all"
```

Comment:
Update browserslist settings to match [Mapbox GL's suggestion](https://docs.mapbox.com/mapbox-gl-js/guides/install/#targeting-transpilation-to-es6-with-browserslist). That means adding:
```
"not ie 11",
"not chrome < 51",
"not safari < 10"
```